weeks/week01/section03.md

```diff
@@ -113,19 +125,36 @@ Write a while loop that simulates retrying a test case up to 5 times until it pa
 - `break`: Exits the loop entirely.
 - `continue`: Skips the current iteration and moves to the next one.
 
+
 #### Examples:
 
    ```python
    # Breaking out of a loop if a condition is met
    test_cases = ["Test1", "Test2", "Test3", "Test4"]
-   
+
    for test in test_cases:
        if test == "Test3":
            print(f"{test} failed, stopping further tests.")
            break
        print(f"Running {test}")
    ```
 
-#### Hands-On Task:
 
-Write a loop that iterates over test cases. If the current test case fails, break out of the loop; otherwise, print the result.
+#### Practical Exercises (Drills):
+
+1. Create a script that uses arithmetic operators to calculate the percentage of passed and failed tests, then use comparison and logical operators to check if the tests meet a predefined success rate, given:
+- Percentage of passed is calculated by dividing the number of passed tests by the total number of tests and then multiplying by 100.
+- Percentage of failed is calculated by subtracting 100 by percentage of passed tests.
+- To check if the percentage of passed tests meets the success rate threshold, just check if the percentage of passed is greater than or equals to a success rate threshold defined earlier.
+
+2. Write a loop that iterates over test cases. If the current test case fails, break out of the loop; otherwise, print the result.
+
+3. Write a script that checks if a user is eligible for a special test case discount:
+- The user must be a registered tester and must have run at least 5 tests in the last week.
+- If the user meets both conditions, print "Eligible for discount".
+- If the user is registered but hasn’t run 5 tests, print "Not enough tests for discount".
+- If the user is not registered, print "Not eligible for discount".
+
+3. Create a list of test results (e.g., [True, False, True, False]). Write a `for` loop that prints `"Test Passed"` for `True` values and `"Test Failed"` for `False` values.
```

Comment:
It would be but we already are talking about the list in this section. We didn't go into details yet. It is okay as during the explanation of loops, there are several examples of lists. See that the exercise shows an example of the list. I think it is fine to have it. What do you think?